### PR TITLE
fixes #74

### LINKED ISSUE #74: Cloudless client configuration
### Case
I have a project with reactive feign clients. Some clients need to be configured in a cloud mode, so I put `feign-reactor-cloud` in the classpath. But several clients are external APIs, so these clients need to be configured in cloudless mode.
###  Expected behavior
In openfeign this case solved by `url` attribute in `@FeignClient`. If `url` is present then load balancer doesn't apply to this client
### Actual behavior
In reactivefeign load balancer applied to these clients
if I use `reactive.feign.ribbon.enabled` and `reactive.feign.hystrix.enabled` it disables ribbon and hystrix globally.

### PR DESCRIPTION
**Pull request by KuliginStepan**
Possible solution to fix #74. The solution is similar to openfeign

### CODE REVIEW

File: feign-reactor-spring-configuration/src/main/java/reactivefeign/spring/config/ReactiveFeignClientFactoryBean.java [hunk 1]
```diff
@@ -287,6 +288,9 @@ public Object getObject() throws Exception {
 			} else {
 				url = this.url;
 			}
+			if (builder instanceof CloudReactiveFeign.Builder) {
```
Comment (tdanylchuk) [inline feign-reactor-spring-configuration/src/main/java/reactivefeign/spring/config/ReactiveFeignClientFactoryBean.java]: we need to add tests in order to verify that issue is fixed and won't be broken in future

**Comment by KuliginStepan** [comment]
Feign clients who use ribbon + hystrix are usually clients for internal services. Feign clients without ribbon + hystrix (without `feign-reactor-cloud` in classpath) are usually clients for external services. While we are not able to disable hystrix separately for each client, I would expect the shutdown of both histrix and the ribbon when using the url attribute.
Also, the problem of hystrix is ​that it is impossible to set one timeout for the whole client​ since the default timeout in hystrix is ​​usually set up taking into account internal services and does not take into account the work of external services. Therefore, in the case of external services, it is more convenient to use the feign native timeouts.

**Comment by kptfh** [comment]
@KuliginStepan Hi. As for now I can see that if url is present in ReactiveFeignClient annotation.
And if it has been set then ribbon is not configured.
[ReactiveFeignClientFactoryBean](https://github.com/Playtika/feign-reactive/blob/develop/feign-reactor-spring-configuration/src/main/java/reactivefeign/spring/config/ReactiveFeignClientFactoryBean.java#L285)
Can you check it from your side so I can close this issue?

**Comment by KuliginStepan** [comment]
@kptfh Hi. It seems nothing has been changed here, so the issue is still present

**Comment by gmaiztegi** [comment]
Hi everyone. We are having the same issue in a project. We use the blocking FeignClient extensively in our projects and we get Ribbon disabled when we add the URL.

This does not happen with the reactive Feign client, but I tried @KuliginStepan's solution and does its job seamlessly.

**Pull request closed without merging**